.travis.yml

```diff
@@ -102,11 +102,10 @@ matrix:
         # flag to make sure no files are being written to the temporary install
         # location during testing.
         - os: linux
-          env: PYTHON_VERSION=3.5 SETUP_CMD='test --remote-data=astropy --readonly'
+          env: PYTHON_VERSION=3.7 SETUP_CMD='test --remote-data=astropy --readonly'
```

Comment:
With this change, I'm not sure we have a `3.5` test left...

Reply:
the one above with np 1.13 looks like another py3.5 one, although we don't seem to have a py3.6 tests now

Reply:
Default Python is still 3.6, so pretty sure there are those runs in there but let me check that log and see.

Reply:
oh, sure never mind me, the appveyor diff below mislead me (also, maybe I should not comment on PRs before having the first coffee of the day...)